.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["2.5", "2.6", "2.7", "3.0", "3.1", ruby-head, jruby-9.2, jruby-9.3]
+        rubocop_version: ["0.86", "1.20"]
```

Comment:
If we are only testing these two versions anyway, perhaps we should delete the two gemfiles in `gemfiles/rubocop_0.53.gemfile` and `gemfiles/rubocop_0.81.gemfile`. What do you think @mikegee ?